src/EditView.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function EditView({onValueChange, ...rest}) {
+  /************************************
+   * Render
+   ************************************/ 
+
+  return (
+     <textarea style={{height: 'inherit'}} {...rest} onChange={onValueChange}></textarea>
```

Comment:
Any reason why you're setting the styling here as opposed to CSS?

Reply:
The height of the 'EditMode' will be determined by the user passing down a height. This text area will calculate the height based off that passed in height (will be shorter at some point). Since it's dynamic i should be doing this styling inline. 

Also from what i've seen these npm packages don't usually use style sheets. Need to determine the best way to do this. Might make the most sense to keep all functionality on a component since it's the one thing being imported